lib/inspec_tools/csv.rb

```diff
@@ -74,7 +74,11 @@ def parse_controls(prefix)
         control['tags']['nist'] = nist unless nist.nil? || nist.include?(nil)
         @mapping['control.tags'].each do |tag|
           if tag.first == 'cci'
-            control['tags'][tag.first] = cci_number
+            if cci_number.is_a? String
```

Comment:
I think it would be better if we check if the cci_number is an Array. If it is not an array let's put it in one.

Reply:
> I think it would be better if we check if the cci_number is an Array. If it is not an array let's put it in one.

Sure thing, I'll make that change.